_posts/2021-10-29-moving-from-opensource-elasticsearch-to-opensearch.markdown

```diff
@@ -7,37 +7,38 @@ date: 2021-10-29
+  description: "Learn how to move from open source Elasticsearch to OpenSearch, and why you want to move now."
 ---
-As OpenSearch is now shipping out generally available releases, I wanted to provide an overview of how moving from open source Elasticsearch to OpenSearch looks like, and help you prepare for the move. 
+*Blog refreshed for technical accuracy on 11 Nov 2022*
 
-### What is an open source Elasticsearch to OpenSearch Upgrade? 
+The process of upgrading from open-source Elasticsearch to OpenSearch varies depending on your current version of Elasticsearch, installation type, tolerance for downtime, and cost sensitivity. Rather than recommended steps for every situation, we provide general guidance on the process.
 
-An *upgrade* is moving a cluster to either a new minor version or new major version. Major version upgrades come with breaking changes which is why it should be carefully upgraded via recommended paths. Minor version upgrades are relatively simple.
+This blog post is a refresh of the upgrade process. 
+## What is an open-source Elasticsearch to OpenSearch upgrade? 
 
-When talking about upgrades, settings and indices are upgraded:
+An *upgrade* means moving a cluster to either a new major or latest minor version of the major version. Major version upgrades come with breaking changes, and the upgrade should follow the recommended migration path. Minor version upgrades are relatively simple.
 
-* Cluster Settings - All dynamic settings which are supported in the new cluster will be migrated and the ones which were deprecated will be *archived*.
-* Indices - All compatible indices (i.e. all your data) will be readable/writable by the new software version.
+When upgrading from Elasticsearch to OpenSearch, settings and indexes are upgraded:
 
-Static settings defined in `elasticsearch.yml` / `opensearch.yml` are not moved automatically during an upgrade.
-OpenSearch 1.1.0 is bundled with a new [upgrade tool](https://opensearch.org/docs/latest/upgrade-to/upgrade-to/#upgrade-tool) which connects to an Elasticsearch cluster and moves static settings which are defined in `elasticsearch.yml` to `opensearch.yml` automatically.
+* Cluster settings: All dynamic settings that are supported in the new cluster will be migrated, and the ones that were deprecated will be archived.
+* Indexes: All compatible indexes (that is, all your data) will be readable/writable by the new software version.
 
-Yay! Now you know what an upgrade is, lets buckle up for the upgrade.
+Static settings defined in the  `opensearch.yml` or `elasticsearch.yml` files are not moved automatically during an upgrade. Each version of OpenSearch is bundled with the [`opensearch-upgrade`](https://opensearch.org/docs/latest/upgrade-to/upgrade-to/#upgrade-tool) tool. The tool connects to an Elasticsearch cluster and moves the static settings. The cluster and the settings are defined in the `elasticsearch.yml` to `opensearch.yml` files automatically.
```

Comment:
Also the upgrade tool, migrates the keystore secrets to opensearch if the customer has any. We could add that as well.